documentation/src/main/asciidoc/server_guide/management.adoc

```diff
@@ -65,3 +65,10 @@ In domain mode the port will be the port of the Remoting connector for the Serve
 </subsystem>
 
 ----
+
+=== Jolokia
+
+You can also expose JMX beans using link:https://jolokia.org/[Jolokia]. In order to do this, just run the server with
+additional parameter `--jmx`, for example: `./standalone.xml -c cloud.xml --jmx`. Jolokia configuration is stored in
+`agent.conf` file. It is possible to override this file by specifying it after `--jmx` parameter. For example:
+`./standalone.sh -c cloud.xml --jmx my-config.properties`.
```

Comment:
Please add a quick way to check if jolokia is up and running, via ```curl http://127.0.0.1:8778/jolokia/version```

Reply:
Jolokia spits a startup message to the logs... Do we need to mention additional check here?

Reply:
It won't hurt to add a simple check :)